### PR TITLE
Backslash/Double-quote Fixes

### PR DESCRIPTION
**Pull request by jucarson**
CUS-3391
CUS-3397
CUS-3399

Add escape character before special characters (backslash and double quotes.)

Added CHECKFORINVALIDCHARS procedure : CD Renew
Updated CHECKFORINVALIDCHARS procedure: M2M and Open sub account.

### CODE REVIEW

File: m2m-transfers/V3/REPWRITERSPECS/BANNO.M2MTRANSFERS.V3.POW [hunk 1]
```diff
@@ -3354,21 +3357,21 @@ END [PROCEDURE]
 
 PROCEDURE CHECKFORINVALIDCHARS
```
Comment (tkainz) [inline m2m-transfers/V3/REPWRITERSPECS/BANNO.M2MTRANSFERS.V3.POW:3358]: I think you'll also want to be checking for invalid characters when pulling the saved name from the external account records.

**Review by tkainz — CHANGES_REQUESTED** [review]
Just 1 item to check on

File: open-subshare/REPWRITERSPECS/BANNO.NEWSUBCREATE.V1.POW [hunk 2]
```diff
@@ -4114,7 +4116,9 @@ END
 
 PROCEDURE CHECKFORINVALIDCHARS
```
Comment (tkainz) [inline open-subshare/REPWRITERSPECS/BANNO.NEWSUBCREATE.V1.POW:4117]: Didn't see where the value of the group names were being checked before being passed back to the UX. Last I checked, the CONFIG program allowed the use of invalid chars when running the config program and defining a group name.

File: m2m-transfers/V3/REPWRITERSPECS/BANNO.M2MTRANSFERS.V3.POW (hunk 1 above)
Comment (jucarson) [reply m2m-transfers/V3/REPWRITERSPECS/BANNO.M2MTRANSFERS.V3.POW:3360]: Added.

**Review by rrobison44 — APPROVED** [review]

**Review by tkainz — APPROVED** [review]

**Review by jordan-keenan — APPROVED** [review]
@jucarson Code changes look great! I love the backslash escape character solution

**Pull request merged**